sync/v1.proto

```diff
@@ -878,12 +947,18 @@ message RuleDownloadRequest {
   string machine_id = 2 [json_name = "machine_id"];
 }
 
+// Only one of `rules` or `file_access_watch_items` should be populated per
+// response. Avoiding using a `onof` because it cannot be used with repeated
+// fields.
```

Comment:
I'm not sure this should be the case - it may be that a server wants to separate them and that's fine, but I think the client should handle receiving both responses in a single message.